git-extra/git-sdk.sh

```diff
@@ -20,3 +20,52 @@ case "$rootdir" in
 	fi
 	;;
 esac
+
+
+sdk () {
+    case "$1" in
+    help|--help|-h)
+        echo "The '$0' shell function helps you to get up and running
```

Comment:
Maybe `echo >&2 ...`? I am quite used to help messages going to stderr...

Reply:
Or even `cat >&2 <<-EOF` (which has the advantage that the here-doc can be indented).